Tests/test_bgzf.py

```diff
@@ -416,6 +416,29 @@ def test_write_tell(self):
 
         h.close()
 
+    def test_bug_1701(self):
```

Comment:
Thanks for writing this test. I've had this in the back of my mind for almost a year and always procrastinated. This is really helpful!